Gemfile

```diff
@@ -1,56 +1,60 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
-ruby "2.2.0"
+ruby '2.2.0'
 
-gem "airbrake"
-gem "bourbon", "~> 3.2.1"
-gem "coffee-rails", "~> 4.1.0"
-gem "delayed_job_active_record"
-gem "email_validator"
-gem "flutie"
-gem "high_voltage"
-gem "i18n-tasks"
-gem "jquery-rails"
-gem "neat", "~> 1.5.1"
-gem "newrelic_rpm"
-gem "normalize-rails", "~> 3.0.0"
-gem "pg"
-gem "rack-timeout"
-gem "rails", "4.2.0"
-gem "recipient_interceptor"
-gem "refills"
-gem "sass-rails", "~> 5.0"
-gem "simple_form"
-gem "title"
-gem "uglifier"
-gem "unicorn"
+gem 'bourbon', '~> 3.2.1'
+gem 'coffee-rails', '~> 4.1.0'
+gem 'delayed_job_active_record'
+gem 'email_validator'
+gem 'flutie'
+gem 'high_voltage'
+gem 'jbuilder'
+gem 'jquery-rails'
+gem 'neat', '~> 1.5.1'
+gem 'normalize-rails', '~> 3.0.0'
+gem 'oj'
+gem 'pg'
+gem 'rack-timeout'
+gem 'rails', '4.2.0'
+gem 'recipient_interceptor'
+gem 'refills'
+gem 'sass-rails', '~> 5.0'
+gem 'simple_form'
+gem 'title'
+gem 'uglifier'
+gem 'unicorn'
 
 group :development do
-  gem "spring"
-  gem "spring-commands-rspec"
-  gem "web-console"
+  gem 'binding_of_caller'
+  gem 'spring'
+  gem 'spring-commands-rspec'
+  gem 'web-console'
 end
 
 group :development, :test do
-  gem "awesome_print"
-  gem "bundler-audit", require: false
-  gem "byebug"
-  gem "dotenv-rails"
-  gem "factory_girl_rails"
-  gem "pry-rails"
-  gem "rspec-rails", "~> 3.0.0"
+  gem 'awesome_print'
+  gem 'bundler-audit', require: false
+  gem 'byebug'
+  gem 'dotenv-rails'
+  gem 'factory_girl_rails'
+  gem 'pry-rails'
+  gem 'rspec-rails', '~> 3.0.0'
 end
 
 group :test do
-  gem "capybara-webkit", ">= 1.2.0"
-  gem "database_cleaner"
-  gem "formulaic"
-  gem "launchy"
-  gem "shoulda-matchers", require: false
-  gem "timecop"
-  gem "webmock"
+  gem 'capybara-webkit', '>= 1.2.0'
+  gem 'database_cleaner'
+  gem 'faker'
+  gem 'formulaic'
+  gem 'launchy'
+  gem 'shoulda-matchers', require: false
+  gem 'timecop'
+  gem 'webmock'
 end
 
 group :staging, :production do
-  gem "rails_stdout_logging"
+  gem 'airbrake'
+  gem 'newrelic_rpm'
+  gem 'rails_stdout_logging'
```

Comment:
FYI, `rails_12factor` includes `rails_stdout_logging` so can omit `rails_stdout_logging` if you want.